.travis.yml

```diff
@@ -5,7 +5,15 @@ _environments:
   - &gcc-warnings
     - CFLAGS="-std=c89 -Wall -Wextra -pedantic"
 
+_scripts:
+  - &install_cmake35
+    - >
+        wget https://cmake.org/files/v3.5/cmake-3.5.2-Linux-x86_64.tar.gz &&
+        sudo tar xfC $(basename $_) / --strip-components=1
+
```

Comment:
Right -- speed is a problem, which is why I set those two tests in `allow_failures`, and set `fast_finish` to true. This has the effect that fast, container-based builds determine the global result of the test, and doesn't undermine the ability to get builds results fast -- slower builds can work in the background without clogging the job queue.